docs/user-guide/upgrade.md

```diff
@@ -4,7 +4,42 @@
 
 ## Release 2.0.0-alpha (not released yet)
 
-With release 2.0.0 we will use .NET 6 for the starter kit. Our docker images will be based on Debian 11 (Bullseye). Please make sure that if you plan to use our Docker images, you use Debian 11 instead of Debian 10. If you plan to use a Debian 10-based OS, you will need to build a custom Docker image.
+With release 2.0.0 we will use .NET 6 for the starter kit. Our docker images will be based on Debian 11 (Bullseye). Please make sure that if you plan to use our Docker images, you use a Debian 11-based OS instead of Debian 10. If you plan to use a Debian 10-based OS, you will need to build a custom Docker image.
```

Comment:
```suggestion
With release 2.0.0 we will use .NET 6 for the starter kit. Our docker images will be based on Debian 11 (Bullseye). Please make sure that if you plan to use our Docker images, you use a Debian 11-based OS instead of Debian 10. If you plan to use a Debian 10-based OS, you might need to build a custom Docker image.
```

Are we sure the issue is on all version of debian, e.g. also x86 systems?

Reply:
I think we tried to check if the issue was there with x86/x64 too and it seemed to me there was no issue.

@bastbu what about double checking by creating a Debian 10 VM on Azure and trying to run on it the x64 image based on Debian 11 of our LNS?

Reply:
I tested it on a AMD64 Debian image, it is not an issue there. The Debian 11 image runs on Debian 10 AMD64. What do we want to do with that info?

Reply:
Let's specify "if you plan to use a Debian 10-based OS on an ARM device,..."